pkgs/tools/text/rst2html5/default.nix

```
@@ -9,6 +9,7 @@ in python3Packages.buildPythonPackage {
 
   src = python3Packages.fetchPypi {
     inherit pname version;
+    format = "wheel";
```

```suggestion
    inherit pname version format;
```

Seriously, I am shamed.